bootstrap.sandboxnet

```diff
@@ -6,7 +6,7 @@ REPO_DIR=/opt/rchain-testnet-node
 GIT_CRYPT_KEY_FILE=/root/rchain-sre-git-crypt-key
 SETUP_SCRIPT_URL=https://raw.githubusercontent.com/rchain/rchain-testnet-node/master/update-run-setup
 PROFILE=sandboxnet
-REPO_BRANCH=dev
+REPO_BRANCH=deploy-propose
```

Comment:
Is this supposed to be changed to `dev` before merge? Please also squash other adequate groups of your commits (e.g. `XXX` with something else).

Reply:
This is a separate commit so that I can remove it just before merging.